cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -438,11 +438,13 @@ void CheckConfiguredRoundtrip(
 void DoSimpleRoundtrip(const std::shared_ptr<Table>& table, bool use_threads,
                        int64_t row_group_size, const std::vector<int>& column_subset,
                        std::shared_ptr<Table>* out,
-                       const std::shared_ptr<ArrowWriterProperties>& arrow_properties =
-                           default_arrow_writer_properties()) {
+                       const std::shared_ptr<ArrowWriterProperties>&
+                           arrow_writer_properties = default_arrow_writer_properties(),
+                       const ArrowReaderProperties& arrow_reader_properties =
+                           default_arrow_reader_properties()) {
```

Comment:
This new parameter doesn't seem used anywhere below? Why add it?

Reply:
Ugh, nice catch. This comes from my early attempts to do a roundtrip and ended up going unnoticed. Will remove it as well as the other one you mentioned.